tracer/src/Datadog.Trace/Debugger/Instrumentation/MethodDebuggerState.cs

```diff
@@ -50,8 +51,11 @@ internal MethodDebuggerState(string probeId, Scope scope, DateTimeOffset? startT
             HasLocalsOrReturnValue = false;
             SnapshotCreator = new DebuggerSnapshotCreator();
             InvocationTarget = invocationTarget;
+            MethodPhase = EvaluateAt.Entry;
         }
 
+        internal EvaluateAt MethodPhase { get; set; }
```

Comment:
Do we still need this?

Reply:
I tend to think we should remove it, seems strange to store this as state (as opposed to just pass it around as parameter if need-be)